.github/workflows/ci.yaml

```
@@ -418,23 +421,29 @@ jobs:
           EOT
           podman info
 
+      - name: task package
+        if: ${{ matrix.task-name != 'wdio' }}
+        id: package
+        run: |
```

actionlint (1.7.12)

[error] property "taskfile_args" is not defined in object type {skip_docker: number; skip_podman: number}

(expression)